### ISSUE TITLE
Optimized Sitemap Author

### ISSUE THREAD

**Issue by EkoJR** [awesomemotive/all-in-one-seo-pack#2209]
When running qcachegrind on sitemap_authors.xml, I've found the these methods/functions to be the top 3 in time used.

All_in_One_SEO_Pack_Sitemap->get_author_prio_from_posts <cycle 2>
aiosp_common::make_xml_safe <cycle 2>
All_in_One_SEO_Pack_Sitemap->get_author_prio_data <cycle 2>

@wpsmort mentioned a user had reported an issue via email regarding  sitemap_authors.xml.

**Comment by wpsmort** [comment]
This was reported by Belal Zewain by email on 2/14.

I was unable to reproduce this on any other site except the site provided by Belal.  I cloned the site and was able to reproduce this with the default theme and no other plugins enabled.  The site had Sitemap Indexes enabled and pagination set to 500.  All other sitemaps loaded fine, the sitemap-author.xml would not load and was using 358Mb of memory.  Bumping memory allocated to PHP up to 512Mb resulted in the sitemap being able to load.  I tested back through versions to v2.6 and the problem was present with each version so this is not new.
